pkg/zson/docs/spec.md

```diff
@@ -1,29 +1,29 @@
-# zson specification
+# ZSON specification
 
-zson is a format for structured data values, ideally suited for streams
+ZSON is a format for structured data values, ideally suited for streams
 of heterogeneously typed records.
-zson is richly typed and thinner than json.
-Like [newline-delimited json (ndjson)](http://ndjson.org/),
-zson represents a sequence of data objects that can be parsed line by line.
+ZSON is richly typed and thinner than JSON.
+Like [newline-delimited JSON (NDJSON)](http://ndjson.org/),
+ZSON represents a sequence of data objects that can be parsed line by line.
 
-zson strikes a balance between the narrowly typed but flexible ndjson format and
+ZSON strikes a balance between the narrowly typed but flexible NDJSON format and
 a more structured approach like
 [Apache Avro](https://avro.apache.org).
-zson is type rich and
+ZSON is type rich and
 embeds all type/schema in the stream, while having a value syntax
-independent of the schema so it is easy and efficient to parse on the fly
+independent of the schema, making it easy and efficient to parse on the fly
 and mix and match streams from different sources with heterogeneous types.
 Like Avro,
-zson embeds schema information in the data stream but zson schema bindings
+ZSON embeds schema information in the data stream but ZSON schema bindings
 are lighter-weight and specified with a simple integer encoding that
-accompanies each each data value.
+accompanies each data value.
 
-The zson design [is also motivated by](./rationale.md)
-and maintains backward compatibility with the original zeek log format.
+The ZSON design [is also motivated by](./rationale.md)
+and maintains backward compatibility with the original [Zeek ASCII TSV log format](https://docs.zeek.org/en/stable/examples/logs/).
 
-## zson format
+## ZSON format
 
-zson is a utf-8 encoded stream of "lines" where each line is terminated by
+ZSON is a UTF-8 encoded stream of "lines" where each line is terminated by
```

Comment:
Nit: Should it be "by newline" or "by a newline"? I've gone back & forth on this.

Reply:
Definitely "by a newline".